app/controllers/seances_controller.rb

```diff
@@ -1,13 +1,11 @@
 class SeancesController < ApplicationController
-  before_action :set_seance, only: %i[show edit update destroy]
+  include Pundit::Authorization
 
-  # GET /seances or /seances.json
-  def index
-    @seances = Seance.all
-  end
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+  # Probably should move this after implementing errorhandler
```

Comment:
You can already move this to `ApplicationController ` - that way you will have it only once :)